### ISSUE TITLE
Kubernetes provider setup broken for version 1.11

### ISSUE THREAD

**Issue by Pratima** [terraform-aws-modules/terraform-aws-eks#733]
# I have issues

## I'm submitting a...

* [x] bug report
* [ ] feature request
* [x] support request - read the [FAQ](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/faq.md) first!
* [ ] kudos, thank you, warm fuzzy

## What is the current behavior?
Kubernetes provider setup breaks when the provider version is upgraded to v1.11

## If this is a bug, how to reproduce? Please include a code sample if relevant.
Provider configuration
```
provider "kubernetes" {
  host                   = data.aws_eks_cluster.cluster.endpoint
  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
  token                  = data.aws_eks_cluster_auth.cluster.token
  load_config_file       = false
  version                = "~>1.9"
}
```

`terraform init`
![image](https://user-images.githubusercontent.com/914695/74203511-04707000-4cc4-11ea-89e9-5665503bb43f.png)

`terraform plan`
![image](https://user-images.githubusercontent.com/914695/74203529-118d5f00-4cc4-11ea-9391-dcbe6bb1d6ee.png)

## What's the expected behavior?
Plan should work

## Environment details

Terraform v0.12.19
+ provider.aws v2.48.0
+ provider.kubernetes v1.11.0
+ provider.local v1.4.0
+ provider.null v2.1.2
+ provider.random v2.2.1
+ provider.template v2.1.2

* OS: Darwin

**Comment by mcavoyk** [comment]
The Terraform provider for Kubernetes mistakenly had a breaking change in their most recent minor version, v1.11.0 See https://github.com/terraform-providers/terraform-provider-kubernetes/issues/759 for more details.

For know the best solution in the case of this module is just to pin to 1.10.0:

```
provider "kubernetes" {
  host                   = data.aws_eks_cluster.cluster.endpoint
  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
  token                  = data.aws_eks_cluster_auth.cluster.token
  load_config_file       = false
  version                = "~>1.10.0"
}
```

**Comment by TBeijen** [comment]
I originally also used the above syntax but that gave me provider version `1.11.0`. So use:
```
version                = "1.10.0"
```

**Comment by Krishna1408** [comment]
I got the same issue. To fix it I rolled back to version `1.10.0`. Hope it gets fixed soon.

**Comment by BossColo** [comment]
Kubernetes provider v1.11.1 fixes this issue.
